.gitignore

```diff
@@ -51,3 +51,4 @@ source/locale/*/cldr.dic
 .venv
 nvdaHelper/docs/
 *.pfx
+appveyor-tools/

```

Comment:
What is this used for?

Reply:
For encrypting files for use in appveyor. TBH it didn't end up being used in this case since appveyor has an encrypt variable function on the website which was enough for the API key.